src/Homepage/Logo/Logo.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import './_logo.scss';
+
+
+
+export default class Logo extends Component {
```

Comment:
Since there is no lifecycle hook or state usage in here, let's make this component a function component.